docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
 
-Examples
---------
-Repurposing a node as a dedicated master node (master: true, data: false)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* ``node.master: true``
+* ``node.data: false``
 
-In this example, a former data node is repurposed as a dedicated master node.
-First update the node’s settings to node.master: true and node.data: false in
-its crate.yml config file. Then run the ``crate-node repurpose`` command to
-find and remove excess shard data:
+.. NOTE::
+
+  Before you continue with the next step, you need to stop CrateDB.
+
+Run ``crate-node repurpose`` to find and remove excess shard data:
 
 .. code-block:: console
 
     node$ ./bin/crate-node repurpose
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
-    Found 2 shards in 2 tables to clean up
-    Use -v to see list of paths and tables affected
-    Node is being re-purposed as master and no-data. Clean-up of shard data will be performed.
+    Found 2 shards in 2 tables to clean up.
+    Use -v to see list of paths and tables affected.
+    Node is being repurposed as master and no-data. Clean-up of shard data will
+    be performed.
 
     Do you want to proceed?
 
-    Confirm [y/N] y
-    Node successfully repurposed to master and no-data.
+    Confirm [y/n] y
+    Node successfully repurposed to master and no data.
+
+Repurposing a node as a coordinating-only node
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Repurposing a node as a coordinating-only node (master: false, data: false)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This example shows you how to repurpose a node, that previously held data, as a
+coordinating-only node.
 
-In this example, a node that previously held data is repurposed as a
-coordinating-only node. First update the node’s settings to node.master: false
-and node.data: false in its crate.yml config file. Then run the
-``crate-node repurpose`` command to find and remove excess shard data and table
+First update the settings in the `crate.yml config file` of the node to:
+
+* ``node.master: false``
+* ``node.data: false``
+
+.. NOTE::
+
+  Before you continue with the next step, you need to stop CrateDB.
+
+Run ``crate-node repurpose`` to find and remove excess shard data and table
 metadata:
 
 .. code-block:: console
 
     node$./bin/crate-node repurpose
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
-    Found 2 tables (2 shards and 2 table meta data) to clean up
-    Use -v to see list of paths and tables affected
-    Node is being re-purposed as no-master and no-data. Clean-up of table data will be performed.
+    Found 2 tables (2 shards and 2 table metadata) to clean up.
+    Use -v to see list of paths and tables affected.
+    Node is being repurposed as no-master and no-data. Clean-up of table data
+    is performed.
 
     Do you want to proceed?
 
-    Confirm [y/N] y
+    Confirm [y/n] y
```

Comment:
N is uppercase on purpose, since it is the default option when you just press enter. This is common with command line tools. So it should be Confirm [y/N]